core/proguard-rules.pro

```diff
@@ -9,3 +9,5 @@
     private void onTableUpdate(java.lang.String);
     private void onTransactionCommit(boolean);
 }
+-keep class org.sqlite.** { *; }
+-dontwarn java.sql.JDBCType
```

Comment:
I wonder if we can be more specific here, but doing this appears to be the common approach for `org.sqlite`. If you feel like it, you could try only keeping the classes [mentioned here](https://github.com/xerial/sqlite-jdbc/blob/bd0e99541a70a31da41f7d7c4153e7ac62840cad/src/main/java/org/sqlite/core/NativeDB.c#L443-L498). But if it's too much effort or breaks the tests, I'd say don't worry.

Reply:
Alright, then let's keep it as is. Probably better anyway because we'd have to maintain these definitions for possible future updates to that package.